Fixed typos in readme.md file and added CONTRIBUTING.md file.

INNOVATIVEGAMER opened the pull request:
### Description

1.Fixed the contributing spelling
2.Added contrinuting.md file
3.Added the contributing guidelines link

Fixes #64 

### Type of Change:
- Documentation


### Checklist:
- My PR follows the style guidelines of this project
- I have made corresponding changes to the documentation

INNOVATIVEGAMER committed:
Fixes issue #64

1.Fixed the contributing spelling
2.Added contrinuting.md file
3.Added the contributing guidelines link

INNOVATIVEGAMER commented:
@isabelcosta Please review this file.
Do the documentation PR also needs 2 reviews ? :confused:

isabelcosta commented:
@INNOVATIVEGAMER yes :) If you could reach out to the community to get more reviews that would be amazing!

isabelcosta reviewed (dismissed):
Thank you for this contribution @INNOVATIVEGAMER 🙌

devkapilbansal commented:
@INNOVATIVEGAMER we already have a commit message guideline. Please follow it. Feel free to change the title to PR to be more descriptive.

devkapilbansal commented:
Also, issue #64 is not related to adding a contribution guidelines file. Please edit the issue for every change this PR contains.

isabelcosta commented:
Agree with @devkapilbansal points :) thank you for providing your input!

gaurivn commented on .github/CONTRIBUTING.md:
```diff
@@ -0,0 +1,22 @@
+# Contributing Guidelines
+
+-   You can join us on [AnitaB.org Open Source Zulip](https://anitab-org.zulipchat.com/). Each active repo has its own stream to direct questions to (for example #powerup or #portal). Bridge in Tech stream is _blank_.
```
Pls make the necessary changes - 
1. Bridge in Tech should be Open Source Programs instead.
2. Add "Open Source Programs stream is [#open-source-progs](https://anitab-org.zulipchat.com/#streams/237907/open-source-progs)." instead.

Thank you, do make necessary changes in the issue  - add contributing.md

INNOVATIVEGAMER committed:
Fixes links and mentions in CONTRIBUTING.md

1.Links to the issues and commit message style are changed to open-source-programs-web
2.Changed hastag of zulip community for open source progs.

INNOVATIVEGAMER commented:
@gaurivn Fixed the changes you mentioned in the CONTRIBUTING.md file.

INNOVATIVEGAMER commented:
@isabelcosta and @devkapilbansal Fixed the changes you mentioned. 
There are 2 commits now.
You can squash and merge them as one commit. I will reset my fork to head commit of the upstream/develop.

Sorry for the commits title , I am new to this repo and as this is just a typo fix in readme so silly mistakes happened.
Thank you !

gaurivn commented on .github/CONTRIBUTING.md:
```diff
@@ -1,9 +1,9 @@
 # Contributing Guidelines
 
--   You can join us on [AnitaB.org Open Source Zulip](https://anitab-org.zulipchat.com/). Each active repo has its own stream to direct questions to (for example #powerup or #portal). Bridge in Tech stream is _blank_.
+-   You can join us on [AnitaB.org Open Source Zulip](https://anitab-org.zulipchat.com/). Each active repo has its own stream to direct questions to (for example #powerup or #portal). Open Source Programs stream is _#open-source-progs_.
```
#open-source-progs - missing link to Open Source Programs Zulip stream https://anitab-org.zulipchat.com/#streams/237907/open-source-progs  (as in markdown - [ name ] ( link )   ), no need of italics

Pls add the link, thank you

INNOVATIVEGAMER committed:
Docs : Added missing link for the zulip open source programs community

INNOVATIVEGAMER commented:
@gaurivn  Fixed the link to zulip community.
Now this typos making me crazy. 😆 
Fixed all or any other changes required ?

gaurivn reviewed (approved):
All good, Thank you for your contribution :confetti_ball: :tada:

INNOVATIVEGAMER commented:
@isabelcosta and @devkapilbansal Fixed the changes you mentioned. 
There are 3 commits now.
You can squash and merge them as one commit. 
I will reset my fork to head commit of the upstream/develop on my side for future commits so no past commits will be included in other PRs

Sorry for the commits title , I am new to this repo and as this is just a typo fix in readme so silly mistakes happened.
Thank you !

isabelcosta reviewed (approved):
Awesome! 👏 thank you for addressing the additional code review comments :)

isabelcosta commented:
Thank you so much for your inputs @devkapilbansal @gaurivn ! 🎉 
I really appreciate it! 💪
